packages/blockchain-wallet-v4-frontend/src/components/Navbar/AppSwitcher.tsx

```diff
@@ -57,20 +57,23 @@ const IconWrapper = styled.div`
   background: ${(props) => props.theme.white};
   cursor: pointer;
   transition: all 0.3s;
-  &.active,
   &:hover {
-    border: 1px solid ${(props) => props.theme.blue600};
-    background: ${(props) => props.theme.blue000};
-    * {
-      fill: ${(props) => props.theme.blue600};
-    }
-  }
-  &.nfts {
-    border: 1px solid ${(props) => props.theme.purple600};
-    background: ${(props) => props.theme.purple000};
-    * {
-      fill: ${(props) => props.theme.purple600};
-    }
+    ${(props) =>
+      props.nfts === 'nfts'
+        ? css`
+            border: 1px solid ${(props) => props.theme.purple600};
+            background: ${(props) => props.theme.purple000};
+            * {
+              fill: ${(props) => props.theme.purple600};
+            }
+          `
+        : css`
+            border: 1px solid ${(props) => props.theme.blue600};
+            background: ${(props) => props.theme.blue000};
+            * {
+              fill: ${(props) => props.theme.blue600};
+            }
+          `}
```

Comment:
whats the purpose of this change?

Reply:
Can you create a scam page ?

Reply:
So its to have it behave same on wallet page and nft page, on hover should light it up but hover away back to grey.